wls-ergebnismeldung-service/src/main/resources/db/migrations/oracle/V1_0__createTableStatus.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE Status
+(
+    wahlid                           VARCHAR(1024) NOT NULL,
+    wahlbezirkid                     VARCHAR(1024) NOT NULL,
+
+    schnellmeldungValidierungsstatus VARCHAR(255)  NOT NULL,
```

Comment:
we will check it separatly cause it will affect multiple services: #623 . The idea looks great cause it will increase data consistency on database level.